modules/test/playwright/tests/frontend-data-set-views-web/sorting.spec.ts

```diff
@@ -168,3 +172,58 @@ test.describe('Configure sorting in Data Set Manager', () => {
 		});
 	});
 });
+
+test.describe('Sorting Dropdown in Data Set Application Page', () => {
+	test('When sorting configuration has no labels defined, the order dropdown is not displayed @LPD-19503', async ({
+		dataSetsPage,
+		page,
+	}) => {
+		await dataSetsPage.goto();
+
+		await expect(
+			page.getByRole('button', {name: 'Order'})
+		).not.toBeVisible();
+	});
+});
+
+test.describe('Sorting Dropdown in Data Set Fragment', () => {
```

Comment:
A common pattern we do in DSM tests is to have two types of tests, one for DSM and one for fragment, allowing separate configurations. See for example [here](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-views-web/settings.spec.ts#L197-L205). In this PR, `isolatedLayoutTest` fixture is unnecessary for DSM, but it will still create a new site and destroy it after.